// @aws-sdk/client-eventbridge/dist-es/models/models_0.js
import { __assign, __extends } from "tslib";
import { EventBridgeServiceException as __BaseException } from "./EventBridgeServiceException";
var ConcurrentModificationException = (function (_super) {
    __extends(ConcurrentModificationException, _super);
    function ConcurrentModificationException(opts) {
        var _this = _super.call(this, __assign({ name: "ConcurrentModificationException", $fault: "client" }, opts)) || this;
        _this.name = "ConcurrentModificationException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, ConcurrentModificationException.prototype);
        return _this;
    }
    return ConcurrentModificationException;
}(__BaseException));
export { ConcurrentModificationException };
var InternalException = (function (_super) {
    __extends(InternalException, _super);
    function InternalException(opts) {
        var _this = _super.call(this, __assign({ name: "InternalException", $fault: "server" }, opts)) || this;
        _this.name = "InternalException";
        _this.$fault = "server";
        Object.setPrototypeOf(_this, InternalException.prototype);
        return _this;
    }
    return InternalException;
}(__BaseException));
export { InternalException };
var InvalidStateException = (function (_super) {
    __extends(InvalidStateException, _super);
    function InvalidStateException(opts) {
        var _this = _super.call(this, __assign({ name: "InvalidStateException", $fault: "client" }, opts)) || this;
        _this.name = "InvalidStateException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, InvalidStateException.prototype);
        return _this;
    }
    return InvalidStateException;
}(__BaseException));
export { InvalidStateException };
var OperationDisabledException = (function (_super) {
    __extends(OperationDisabledException, _super);
    function OperationDisabledException(opts) {
        var _this = _super.call(this, __assign({ name: "OperationDisabledException", $fault: "client" }, opts)) || this;
        _this.name = "OperationDisabledException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, OperationDisabledException.prototype);
        return _this;
    }
    return OperationDisabledException;
}(__BaseException));
export { OperationDisabledException };
var ResourceNotFoundException = (function (_super) {
    __extends(ResourceNotFoundException, _super);
    function ResourceNotFoundException(opts) {
        var _this = _super.call(this, __assign({ name: "ResourceNotFoundException", $fault: "client" }, opts)) || this;
        _this.name = "ResourceNotFoundException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, ResourceNotFoundException.prototype);
        return _this;
    }
    return ResourceNotFoundException;
}(__BaseException));
export { ResourceNotFoundException };
export var ApiDestinationState;
(function (ApiDestinationState) {
    ApiDestinationState["ACTIVE"] = "ACTIVE";
    ApiDestinationState["INACTIVE"] = "INACTIVE";
})(ApiDestinationState || (ApiDestinationState = {}));
export var ApiDestinationHttpMethod;
(function (ApiDestinationHttpMethod) {
    ApiDestinationHttpMethod["DELETE"] = "DELETE";
    ApiDestinationHttpMethod["GET"] = "GET";
    ApiDestinationHttpMethod["HEAD"] = "HEAD";
    ApiDestinationHttpMethod["OPTIONS"] = "OPTIONS";
    ApiDestinationHttpMethod["PATCH"] = "PATCH";
    ApiDestinationHttpMethod["POST"] = "POST";
    ApiDestinationHttpMethod["PUT"] = "PUT";
})(ApiDestinationHttpMethod || (ApiDestinationHttpMethod = {}));
export var ArchiveState;
(function (ArchiveState) {
    ArchiveState["CREATE_FAILED"] = "CREATE_FAILED";
    ArchiveState["CREATING"] = "CREATING";
    ArchiveState["DISABLED"] = "DISABLED";
    ArchiveState["ENABLED"] = "ENABLED";
    ArchiveState["UPDATE_FAILED"] = "UPDATE_FAILED";
    ArchiveState["UPDATING"] = "UPDATING";
})(ArchiveState || (ArchiveState = {}));
export var AssignPublicIp;
(function (AssignPublicIp) {
    AssignPublicIp["DISABLED"] = "DISABLED";
    AssignPublicIp["ENABLED"] = "ENABLED";
})(AssignPublicIp || (AssignPublicIp = {}));
export var ReplayState;
(function (ReplayState) {
    ReplayState["CANCELLED"] = "CANCELLED";
    ReplayState["CANCELLING"] = "CANCELLING";
    ReplayState["COMPLETED"] = "COMPLETED";
    ReplayState["FAILED"] = "FAILED";
    ReplayState["RUNNING"] = "RUNNING";
    ReplayState["STARTING"] = "STARTING";
})(ReplayState || (ReplayState = {}));
var IllegalStatusException = (function (_super) {
    __extends(IllegalStatusException, _super);
    function IllegalStatusException(opts) {
        var _this = _super.call(this, __assign({ name: "IllegalStatusException", $fault: "client" }, opts)) || this;
        _this.name = "IllegalStatusException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, IllegalStatusException.prototype);
        return _this;
    }
    return IllegalStatusException;
}(__BaseException));
export { IllegalStatusException };
var LimitExceededException = (function (_super) {
    __extends(LimitExceededException, _super);
    function LimitExceededException(opts) {
        var _this = _super.call(this, __assign({ name: "LimitExceededException", $fault: "client" }, opts)) || this;
        _this.name = "LimitExceededException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, LimitExceededException.prototype);
        return _this;
    }
    return LimitExceededException;
}(__BaseException));
export { LimitExceededException };
var ResourceAlreadyExistsException = (function (_super) {
    __extends(ResourceAlreadyExistsException, _super);
    function ResourceAlreadyExistsException(opts) {
        var _this = _super.call(this, __assign({ name: "ResourceAlreadyExistsException", $fault: "client" }, opts)) || this;
        _this.name = "ResourceAlreadyExistsException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, ResourceAlreadyExistsException.prototype);
        return _this;
    }
    return ResourceAlreadyExistsException;
}(__BaseException));
export { ResourceAlreadyExistsException };
var InvalidEventPatternException = (function (_super) {
    __extends(InvalidEventPatternException, _super);
    function InvalidEventPatternException(opts) {
        var _this = _super.call(this, __assign({ name: "InvalidEventPatternException", $fault: "client" }, opts)) || this;
        _this.name = "InvalidEventPatternException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, InvalidEventPatternException.prototype);
        return _this;
    }
    return InvalidEventPatternException;
}(__BaseException));
export { InvalidEventPatternException };
export var ConnectionAuthorizationType;
(function (ConnectionAuthorizationType) {
    ConnectionAuthorizationType["API_KEY"] = "API_KEY";
    ConnectionAuthorizationType["BASIC"] = "BASIC";
    ConnectionAuthorizationType["OAUTH_CLIENT_CREDENTIALS"] = "OAUTH_CLIENT_CREDENTIALS";
})(ConnectionAuthorizationType || (ConnectionAuthorizationType = {}));
export var ConnectionOAuthHttpMethod;
(function (ConnectionOAuthHttpMethod) {
    ConnectionOAuthHttpMethod["GET"] = "GET";
    ConnectionOAuthHttpMethod["POST"] = "POST";
    ConnectionOAuthHttpMethod["PUT"] = "PUT";
})(ConnectionOAuthHttpMethod || (ConnectionOAuthHttpMethod = {}));
export var ConnectionState;
(function (ConnectionState) {
    ConnectionState["AUTHORIZED"] = "AUTHORIZED";
    ConnectionState["AUTHORIZING"] = "AUTHORIZING";
    ConnectionState["CREATING"] = "CREATING";
    ConnectionState["DEAUTHORIZED"] = "DEAUTHORIZED";
    ConnectionState["DEAUTHORIZING"] = "DEAUTHORIZING";
    ConnectionState["DELETING"] = "DELETING";
    ConnectionState["UPDATING"] = "UPDATING";
})(ConnectionState || (ConnectionState = {}));
export var ReplicationState;
(function (ReplicationState) {
    ReplicationState["DISABLED"] = "DISABLED";
    ReplicationState["ENABLED"] = "ENABLED";
})(ReplicationState || (ReplicationState = {}));
export var EndpointState;
(function (EndpointState) {
    EndpointState["ACTIVE"] = "ACTIVE";
    EndpointState["CREATE_FAILED"] = "CREATE_FAILED";
    EndpointState["CREATING"] = "CREATING";
    EndpointState["DELETE_FAILED"] = "DELETE_FAILED";
    EndpointState["DELETING"] = "DELETING";
    EndpointState["UPDATE_FAILED"] = "UPDATE_FAILED";
    EndpointState["UPDATING"] = "UPDATING";
})(EndpointState || (EndpointState = {}));
var ManagedRuleException = (function (_super) {
    __extends(ManagedRuleException, _super);
    function ManagedRuleException(opts) {
        var _this = _super.call(this, __assign({ name: "ManagedRuleException", $fault: "client" }, opts)) || this;
        _this.name = "ManagedRuleException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, ManagedRuleException.prototype);
        return _this;
    }
    return ManagedRuleException;
}(__BaseException));
export { ManagedRuleException };
export var EventSourceState;
(function (EventSourceState) {
    EventSourceState["ACTIVE"] = "ACTIVE";
    EventSourceState["DELETED"] = "DELETED";
    EventSourceState["PENDING"] = "PENDING";
})(EventSourceState || (EventSourceState = {}));
export var RuleState;
(function (RuleState) {
    RuleState["DISABLED"] = "DISABLED";
    RuleState["ENABLED"] = "ENABLED";
})(RuleState || (RuleState = {}));
export var LaunchType;
(function (LaunchType) {
    LaunchType["EC2"] = "EC2";
    LaunchType["EXTERNAL"] = "EXTERNAL";
    LaunchType["FARGATE"] = "FARGATE";
})(LaunchType || (LaunchType = {}));
export var PlacementConstraintType;
(function (PlacementConstraintType) {
    PlacementConstraintType["DISTINCT_INSTANCE"] = "distinctInstance";
    PlacementConstraintType["MEMBER_OF"] = "memberOf";
})(PlacementConstraintType || (PlacementConstraintType = {}));
export var PlacementStrategyType;
(function (PlacementStrategyType) {
    PlacementStrategyType["BINPACK"] = "binpack";
    PlacementStrategyType["RANDOM"] = "random";
    PlacementStrategyType["SPREAD"] = "spread";
})(PlacementStrategyType || (PlacementStrategyType = {}));
export var PropagateTags;
(function (PropagateTags) {
    PropagateTags["TASK_DEFINITION"] = "TASK_DEFINITION";
})(PropagateTags || (PropagateTags = {}));
var PolicyLengthExceededException = (function (_super) {
    __extends(PolicyLengthExceededException, _super);
    function PolicyLengthExceededException(opts) {
        var _this = _super.call(this, __assign({ name: "PolicyLengthExceededException", $fault: "client" }, opts)) || this;
        _this.name = "PolicyLengthExceededException";
        _this.$fault = "client";
        Object.setPrototypeOf(_this, PolicyLengthExceededException.prototype);
        return _this;
    }
    return PolicyLengthExceededException;
}(__BaseException));
export { PolicyLengthExceededException };
export var ActivateEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ApiDestinationFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ArchiveFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CancelReplayRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CancelReplayResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateApiDestinationRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateApiDestinationResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateArchiveRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateArchiveResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionApiKeyAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionBasicAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionBodyParameterFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionHeaderParameterFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionQueryStringParameterFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionHttpParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionOAuthClientRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionOAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateConnectionResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EndpointEventBusFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ReplicationConfigFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PrimaryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var SecondaryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var FailoverConfigFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RoutingConfigFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateEndpointRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateEndpointResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TagFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateEventBusRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreateEventBusResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreatePartnerEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CreatePartnerEventSourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeactivateEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeauthorizeConnectionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeauthorizeConnectionResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteApiDestinationRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteApiDestinationResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteArchiveRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteArchiveResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteConnectionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteConnectionResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteEndpointRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteEndpointResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteEventBusRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeletePartnerEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeleteRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeApiDestinationRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeApiDestinationResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeArchiveRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeArchiveResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeConnectionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionApiKeyAuthResponseParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionBasicAuthResponseParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionOAuthClientResponseParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionOAuthResponseParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionAuthResponseParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeConnectionResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEndpointRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEndpointResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEventBusRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEventBusResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeEventSourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribePartnerEventSourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribePartnerEventSourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeReplayRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ReplayDestinationFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeReplayResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DescribeRuleResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DisableRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EnableRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListApiDestinationsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListApiDestinationsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListArchivesRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListArchivesResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListConnectionsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConnectionFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListConnectionsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEndpointsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EndpointFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEndpointsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEventBusesRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EventBusFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEventBusesResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEventSourcesRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EventSourceFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListEventSourcesResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListPartnerEventSourceAccountsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PartnerEventSourceAccountFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListPartnerEventSourceAccountsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListPartnerEventSourcesRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PartnerEventSourceFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListPartnerEventSourcesResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListReplaysRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ReplayFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListReplaysResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListRuleNamesByTargetRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListRuleNamesByTargetResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListRulesRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RuleFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListRulesResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListTagsForResourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListTagsForResourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListTargetsByRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var BatchArrayPropertiesFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var BatchRetryStrategyFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var BatchParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var DeadLetterConfigFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var CapacityProviderStrategyItemFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var AwsVpcConfigurationFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var NetworkConfigurationFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PlacementConstraintFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PlacementStrategyFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var EcsParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var HttpParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var InputTransformerFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var KinesisParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RedshiftDataParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RetryPolicyFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RunCommandTargetFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RunCommandParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var SageMakerPipelineParameterFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var SageMakerPipelineParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var SqsParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TargetFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ListTargetsByRuleResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutEventsRequestEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutEventsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutEventsResultEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutEventsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutPartnerEventsRequestEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutPartnerEventsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutPartnerEventsResultEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutPartnerEventsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var ConditionFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutPermissionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutRuleRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutRuleResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutTargetsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutTargetsResultEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var PutTargetsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RemovePermissionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RemoveTargetsRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RemoveTargetsResultEntryFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var RemoveTargetsResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var StartReplayRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var StartReplayResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TagResourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TagResourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TestEventPatternRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var TestEventPatternResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UntagResourceRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UntagResourceResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateApiDestinationRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateApiDestinationResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateArchiveRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateArchiveResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionApiKeyAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionBasicAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionOAuthClientRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionOAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionAuthRequestParametersFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateConnectionResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateEndpointRequestFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
export var UpdateEndpointResponseFilterSensitiveLog = function (obj) { return (__assign({}, obj)); };
